resources/model/OpenStudio.idd

```diff
@@ -33367,6 +33367,211 @@ OS:Meter:CustomDecrement,
         \begin-extensible
    A6;  \field Output Variable or Meter Name
 
+OS:Output:EnvironmentalImpactFactors,
+   \memo This is used to Automatically report the facility meters and turn on the Environmental Impact Report calculations
+   \memo for all of the Environmental Factors.
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2 ; \field Reporting Frequency
+       \type choice
+       \required-field
+       \note Timestep refers to the zone Timestep/Number of Timesteps in hour value
+       \note RunPeriod and Environment are the same. Detailed is not a valid choice.
+       \key Timestep
+       \key Hourly
+       \key Daily
+       \key Monthly
+       \key RunPeriod
+       \key Environment
+       \key Annual
+
+OS:EnvironmentalImpactFactors,
+   \memo Used to help convert district and ideal energy use to a fuel type and provide total carbon equivalent with coefficients
+   \memo Also used in Source=>Site conversions.
+   \unique-object
+  A1, \field Handle
+      \type handle
+      \required-field
+  N1, \field District Heating Efficiency
+      \note District heating efficiency used when converted to natural gas
+      \minimum> 0.0
+      \required-field
+  N2, \field District Cooling COP
+      \note District cooling COP used when converted to electricity
+      \units W/W
+      \minimum> 0.0
+      \required-field
+  N3, \field Steam Conversion Efficiency
+      \note Steam conversion efficiency used to convert steam usage to natural gas
+      \minimum> 0.0
+      \required-field
+  N4, \field Total Carbon Equivalent Emission Factor From N2O
+      \units kg/kg
+      \required-field
+  N5, \field Total Carbon Equivalent Emission Factor From CH4
+      \units kg/kg
+      \required-field
+  N6; \field Total Carbon Equivalent Emission Factor From CO2
+      \units kg/kg
+      \required-field
+
+OS:FuelFactors,
+     \memo Provides Fuel Factors for Emissions as well as Source=>Site conversions.
+     \memo OtherFuel1, OtherFuel2 provide options for users who want to create and use
+     \memo fuels that may not be mainstream (biomass, wood, pellets).
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2,  \field Existing Fuel Resource Name
+       \required-field
+       \type choice
+       \key Electricity
+       \key NaturalGas
+       \key FuelOilNo1
+       \key FuelOilNo2
+       \key Coal
+       \key Gasoline
+       \key Propane
+       \key Diesel
+       \key OtherFuel1
```

Comment:
I **removed**   A2,   \field Units of Measure and   N1,   \field Energy per Unit Factor

They are NOT used in the E+ source code.